src/TableFooter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const TableFooter = ({dataArrayLength, currentPage, entriesPerPage, onChange, dataLength}) => {
+  const pageNumbers = []
+  for (let i = 1; i <= Math.ceil(dataArrayLength / entriesPerPage); i++) {
+    pageNumbers.push( i === currentPage ? <li className={`current`} key={`bottom${i}`}>{currentPage}</li> :
+      <li key={`bottom${i}`}><a onClick={() => onChange(i)}>{i}</a></li>)
+  }
+
+  return (
+    <div className="row expanded padding-top-medium">
+      <div className={`small-6 columns`}>
+        {
+          dataArrayLength === 0 ?
+            `There are no results for this query.` :
+            `Showing ${dataArrayLength} out of ${dataLength} experiments.`
+        }
+      </div>
+
+      <div className={`small-6 columns`}>
+        <ul className={`pagination`} style={{textAlign: `right`}}>
+          {[
+            currentPage > 1 ?
+              <li key={`previous`}>
+                <a onClick={() => {currentPage > 1 && onChange(currentPage - 1)}}>
+                  Previous
+                </a>
+              </li> :
+              <li key={`previous`} className={`disabled`}>Previous</li>,
+            pageNumbers,
+            dataArrayLength > currentPage * entriesPerPage ?
+              <li key={`next`}>
+                <a onClick={() => {dataArrayLength >= currentPage * entriesPerPage && onChange(currentPage + 1)}}>
+                  Next
+                </a>
+              </li> :
+              <li key={`next`} className={`disabled`}>Next</li>
+          ]}
+        </ul>
+      </div>
+    </div>
+  )
+}
+
+TableFooter.propTypes = {
+  dataArrayLength: PropTypes.number.isRequired,
+  currentPage: PropTypes.number.isRequired,
+  entriesPerPage: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number
+  ]),
```

Comment:
Why is `entriesPerPage` going to be either a string or a number?

Reply:
Because the options are composed of `All` and `10, 25, 50` etc.

Reply:
What value does the prop entriesPerPage have when you click on *All*?